Topics/User_Experience/Mobile-First_Design.md

```diff
@@ -34,8 +34,12 @@ The reasoning behind this strategy is to promote _progressive enhancement_ — t
 
 1. Content inventory
     * Plan out all the content and elements you need for your web pages
+    * This can be written out on a spreadsheet (Microsoft Excel, Google spreadsheets, etc.), or any text editor
+    * The choice of which platform to use is up to you, but a spreadsheet is recommended for easy organization — here’s an example: https://maadmob.com.au/resources/content_inventory
 2. Visual hierarchy
     * Create a priority list of all the visuals you will include on the page, taking into consideration the size of images, colours, and typography, as these properties can guide the user’s eye to understand the relative significance of the elements
+    * As with step 1, this can be done on a spreadsheet or any text editor — the purpose is just to help you plan out the content you’ll need and their importance, which will make designing the actual page easier and ensure most, if not all, of the content is not overlooked
```

Comment:
Perfect! This now allows students to choose between a more visual or text-based option for their planning!